app/views/search/index.html.erb

```diff
@@ -20,15 +20,25 @@
   <% end %> 
     <div class="row">
       <div class="col">
-        <div class="list-group">
-          <%if @results %>
-            <% @results.each do |result|%>
+        <div class="list-group" id="exact-results">
+          <%if @exact_results.present? %>
```

Comment:
I would like to display a "No matches found." in the following case:
`<%unless @exact_results.present? or @more_results.present? %>`

Reply:
I agree, however this is not part of our ticket and this solution is not sufficient: when a user visits the search page 'No matches found' will be rendered